docs/tutorials/widgets/implementing-an-inline-widget.md

```diff
@@ -14,163 +14,30 @@ You will build a "placeholder" feature that allows the users to insert predefine
 	If you want to see the final product of this tutorial before you plunge in, check out the [demo](#demo).
 </info-box>
 
-## Before you start ⚠️
+## Before you start
 
 This guide assumes that you are familiar with the widgets concept introduced in the {@link tutorials/widgets/implementing-a-block-widget Implementing a block widget} tutorial. The tutorial will also reference various concepts from the {@link framework/architecture/intro CKEditor&nbsp;5 architecture}.
```

Comment:
I wonder whether linking Widget guide wouldn't be better maybe? Will it be sufficient?